Sources/ArcGISToolkit/Components/Form/FormInputs/ComboBoxInput.swift

```diff
@@ -66,12 +63,11 @@ struct ComboBoxInput: View {
     
     /// Creates a view for a combo box input.
     /// - Parameters:
-    ///   - featureForm: The feature form containing the input.
     ///   - element: The input's parent element.
-    ///   - input: The input configuration of the view.
-    init(featureForm: FeatureForm?, element: FieldFormElement, input: ComboBoxFormInput) {
-        self.featureForm = featureForm
+    init(element: FieldFormElement) {
         self.element = element
+        
+        let input = element.input as! ComboBoxFormInput
```

Comment:
Would this be good as a `precondition`? `precondition(element.input is ComboBoxFormInput)` to verify that the force-unwrapping will succeed and potentially give developers more information?

Reply:
We could do that, but seeing as the individual input views aren't public (at least for now) that error would be on our side, right?

I just gave it a try and we'd still need the force-unwrap (the precon. doesn't seem to alleviate that need) so we'd just be adding a line to each init.

That said, still happy to add them in if you like?

Reply:
When the `precondition` fails, does the user see a better message than the 'unwrapping a nil' message? That would be the reason to put it in there

Reply:
The default precondition error would look like:

> Precondition failed

I've added preconditions with a more helpful message:

> Precondition failed: SingleLineTextInput.init(element:) element's input must be TextBoxFormInput.